modules/nw-dns-cache-tuning.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * networking/dns-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-cache-tuning_{context}"]
+= Tuning the CoreDNS cache
+
+You can configure the maximum duration of both successful (positive) or unsuccessful (negative) caching done by CoreDNS. Tuning the duration of caching of DNS query responses can reduce the load for any upstream DNS resolvers.
+
+.Procedure
+
+. Edit the DNS Operator object named `default` by running the following command:
++
+[source,terminal]
+----
+$ oc edit dns.operator.openshift.io/default
+----
+
+. Modify the time-to-live (TTL) caching values:
++
+.Configuring DNS caching
+[source,yaml]
+----
+spec:
```

Comment:
@jldohmann, do we know what spec this is? From the edit command, it's the  object named `default` of the DNS API in the operator.openshift.io group, probably v1, but I'd have to check.

I think this should be included in the YAML, per our guidelines for k8s objects, for clarity.

Reply:
@jboxman-rh do you mean including the `apiVersion` field in the YAML file? I can add that for sure